roles/foreman/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+---
+foreman_container_image: "quay.io/evgeni/foreman-rpm"
```

Comment:
Don't forget to add this to the repo

Reply:
https://github.com/ehelms/foreman-quadlet/pull/20